brats/transformations.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+CHANNELS = 3
```

Comment:
This name is misleading. First I was thinking this is amount of channels. I would call it CHANNEL_IDX or something instead.

Reply:
Changed